bower.json

```diff
@@ -3,34 +3,16 @@
 	"description": "Mimics &lt;input type=&#34;cosmoz-datetime-input&#34;&gt; with brad browser support",
 	"main": "cosmoz-datetime-input.html",
 	"dependencies": {
-		"polymer": "Polymer/polymer#1.9 - 2",
-		"paper-input": "PolymerElements/paper-input#1 - 2"
+		"polymer": "Polymer/polymer#^2.0.0",
```

Comment:
Please specify the versions used when testing (like #^2.6.0) even if it will install the same version, to highlight when we last cared about the version :)

Reply:
Done!